packages/tronwrap/index.js

```diff
@@ -59,6 +50,49 @@ function init(options) {
     options.privateKey
   );
 
+  TronWrap.prototype._options = options;
+
+  TronWrap.prototype._getNetwork = function (callback) {
+    callback && callback(null, options.network_id);
+  }
+
+  TronWrap.prototype._getAccounts = function (callback) {
+
+    const self = this
+
+    return new Promise((accept, reject) => {
+      function cb() {
+        if (callback) {
+          callback(null, self._accounts)
+          accept()
+        }
+        else {
+          accept(self._accounts)
+        }
+      }
+
+      if (self._accounts) {
+        return cb()
+      }
+      self._accounts = [options.from || 'TPL66VK2gCXNCD7EJg9pgJRfqcRazjhUZY'];
+      self._privateKeyByAccount = {}
+      return axios.get(self._options.fullNode + '/admin/accounts-json')
```

Comment:
```suggestion
      return axios.get(self._options.fullNode + '/admin/accounts-json')
```
This is not a fullnode http api. It is returned buy the http-proxy on the same port of the full node, so that TronBox does not need to have another configuration in the `tronbox.js`